wolfcrypt/src/rsa.c

```diff
@@ -3373,7 +3373,23 @@ static int RsaPublicEncryptEx(const byte* in, word32 inLen, byte* out,
                                   pad_value, pad_type, hash, mgf, label,
                                   labelSz, sz);
         }
-    #endif /* WOLFSSL_CRYPTOCELL */
+    #elif defined(WOLFSSL_RENESAS_SCEPROTECT_CRYPTONLY)
+           /* SCE needs warpped key which is passed via
+            * user ctx object of crypt-call back.
+            */
+           #ifdef WOLF_CRYPTO_CB
+            if (key->devId != INVALID_DEVID) {
+                /* SCE supports 1024 and 2048 bits */
+                ret = wc_CryptoCb_Rsa(in, inLen, out,
+                                    outLen, rsa_type, key, rng);
+                if (ret != CRYPTOCB_UNAVAILABLE)
+                    return ret;
```

Comment:
Can we add a debug message here letting users know that fall through happened?

Reply:
Agree to you. This could happen when Rsa key is not neither 1024 nor 2048 bits. I would like to check it and show a debug message in callback routine at renesas_common.c.